.github/workflows/build.yml

```diff
@@ -212,7 +213,7 @@ jobs:
           echo "Labels: $LABELS"
           echo "Target Branch: $TARGET_BRANCH"
 
-          if [ "$LABELCHECK" == "" ] && [ "$TARGET_BRANCH" != "beta" ] && [ "$TARGET_BRANCH" != "release" ]; then
+          if [ "$LABELCHECK" == "" ] && [ "$TARGET_BRANCH" != "beta" ] && [ "$TARGET_BRANCH" != "release" ] && [ "$TARGET_BRANCH" != "LTS2022" ]; then
```

Comment:
It would be nice to clean this up in a similar way. Either replace the multiple `&&` branches with `${{ github.* }}` or check for an LTS prefix like ``[[ ! -z "`echo $TARGET_BRANCH | grep -o '^LTS'`" ]]``